src/jbmc/jbmc_parse_options.cpp

```diff
@@ -154,6 +154,11 @@ void jbmc_parse_optionst::get_command_line_options(optionst &options)
   else
     options.set_option("propagation", true);
 
+  // transform self loops to assumptions
```

Comment:
The documentation for this option in JBMC is not complete. Either remove (not really used anywhere at the moment) or make it complete.

Reply:
That's actually provided via `HELP_BMC` to all front-ends implementing BMC.